nexus-sdk-js/src/Resource/index.ts

```diff
@@ -42,6 +44,20 @@ export interface ResourceResponse extends ResourceResponseCommon {
   '@context': Context;
 }
 
+export const getResource = async (
+  id: string,
+  orgLabel: string,
+  projectLabel: string,
+): Promise<Resource> => {
+  try {
+    const resourceResponse: ResourceResponse = await httpGet(id, false);
+    const resource = new Resource(orgLabel, projectLabel, resourceResponse);
+    return resource;
+  } catch (e) {
+    return e;
```

Comment:
I would either throw an error here or return something more useful. I would also change the function signature to something like `getResource(....): Promise<Resource | ResourceError>`